app/colors/catppuccin.css

```diff
@@ -0,0 +1,200 @@
+/*
+* Vieb - Vim Inspired Electron Browser
+* Copyright (C) 2019-2024 Jelmer van Arnhem
+*
+* This program is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+/*
+ * COLORS
+*/
+:root {
+    /* general */
+    --bg: #11111b;
+    --fg: #cdd6f4;
+    --tab-foreground: #a6adc8;
+    --tab-background: #181825;
+    --tab-suspended: #181825;
```

Comment:
Can we give suspended tabs a different color? I believe this is important information that's not supposed to be gone by using a different colorscheme.